jib-core/src/main/java/com/google/cloud/tools/jib/JibSystemProperties.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018 Google LLC. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib;
+
+/** Names of system properties defined/used by Jib. */
+public class JibSystemProperties {
```

Comment:
I think if we have a class for this, we might as well have it be the class with the getters for the system properties like:

`int getHttpTimeout`
`bool isSerializedExecutionEnabled`
`bool isSendCredentialsOverHttpEnabled`
`bool isUserAgentEnabled`

Reply:
Yeah I started out in this direction but wasn't sure how to deal with `ConfigurationPropertyValidator#checkHttpTimeoutProperty(...)`, since it gets `jib.httpTimeout` as a String first. I guess maybe we could move this method to the new class?

Reply:
Hmm, yea that method can live in this new class.

Reply:
I think you could just expose the property name constants for tests and not have to add more methods just for testing purposes.